README.md

```diff
@@ -18,6 +18,12 @@ npm start
 
 Which will start the web application
 
+### Logging in 
+
+To log in you need to provide an IDP or a WebID.
+The application will detect which one you use and log you in.
+If you use your WebID, the first oidc issuer on your WebID is used when there are multiple. 
```

Comment:
"OIDC" instead of "oidc".

Reply:
I have adjusted this in [commit fbb0a52](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/fbb0a52808e8b4ad65a713b5a74efaa0bf04efa3)